client/extensions/tga-author-profile-fields/src/components/profileText/preview.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import {IPreviewComponentProps} from 'superdesk-api';
+import {FormLabel} from 'superdesk-ui-framework/react';
+
+export class ProfileTextPreview extends React.PureComponent<IPreviewComponentProps> {
+    render() {
+        return (
+            <div className="form__row form__row--small-padding">
+                <FormLabel
+                    text={this.props.field.display_name}
```

Comment:
Custom fields should not render their labels. Labels are supposed to be rendered by parent component that renders custom field preview. It is already working like that with "print preview" aka "full preview".

Reply:
This is needed because without it the preview panel doesn't show the field labels, which makes it difficult to understand what field each data is associated with.

Without Labels:
![image](https://user-images.githubusercontent.com/22425232/204965192-e753ffbf-b128-4338-99f6-c8d3c1c16918.png)

With Labels:
![image](https://user-images.githubusercontent.com/22425232/204965217-a5b7ac93-c47c-4c5b-9c7d-90000b05f3ce.png)

Reply:
I agree that labels are needed in the UI, but they should be added from [client-core side] to preview panel (https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/archive/views/preview.html#L170). [Full preview](https://user-images.githubusercontent.com/2076953/205048247-9461460d-18ff-4e47-afd2-ebb20be0bd06.mp4) for example already adds labels and labels would be duplicated in your case